### ISSUE TITLE
nar list of created package

### ISSUE THREAD

**Issue by thesebas** [h2non/nar#139]
When trying to list some simple created package `nar list nodetests-1.0.0.nar` got this error

```
Package: nodetests-1.0.0.nar
C:\Users\thesebas\AppData\Roaming\npm\node_modules\nar\lib\list.js:83
                                                        size: entries[file.archive].size,
                                                                                   ^

TypeError: Cannot read property 'size' of undefined
    at C:\Users\thesebas\AppData\Roaming\npm\node_modules\nar\lib\list.js:83:35
    at Array.forEach (native)
    at emitEntries (C:\Users\thesebas\AppData\Roaming\npm\node_modules\nar\lib\list.js:77:74)
    at Parse.<anonymous> (C:\Users\thesebas\AppData\Roaming\npm\node_modules\nar\lib\list.js:96:5)
    at emitNone (events.js:86:13)
    at Parse.emit (events.js:185:7)
    at Parse._streamEnd (C:\Users\thesebas\AppData\Roaming\npm\node_modules\nar\node_modules\tar\lib\parse.js:65:6)
    at BlockStream.<anonymous> (C:\Users\thesebas\AppData\Roaming\npm\node_modules\nar\node_modules\tar\lib\parse.js:51:8)
    at emitNone (events.js:86:13)
    at BlockStream.emit (events.js:185:7)
```
seems like `nar.files` here https://github.com/h2non/nar/blob/master/src/list.ls#L50  needs to be filtered because there are `file`s that has no `.archive` field, like this:

```
{ name: '_modules-bindir',
  type: 'binaries',
  links:
   { needle: 'D:\\projects\\nodetests\\node_modules\\needle\\bin\\needle\\needle',
     'needle.cmd': 'D:\\projects\\nodetests\\node_modules\\needle.cmd\\bin\\needle.cmd\\needle.cmd' } }
```

**Comment by h2non** [comment]
Thank you. The element in the map could not exists and that's why we have the error here.

**Comment by h2non** [comment]
Fixed in `nar@0.3.38`. 
Update `nar` running: `npm upgrade -g nar`.
